packages/widgetbook/README.md

```diff
@@ -206,6 +294,89 @@ Device(
 ),
 ```
 
+## `frames`
+
+The `frames` property allows developers to define different ways of how the frame of a device is visualized. The following `WidgetbookFrame`s are defined:
+
+`WidgetbookFrame` | Comment | Is default
+------------ | ------------- | ------------- 
+`WidgetbookFrame.defaultFrame` | The default frame of `Widgetbook` | ✅ | 
+`WidgetbookFrame.noFrame` | No frame - this just shows the use case without any device restrictions | ✅ |
+`WidgetbookFrame.deviceFrame` | A frame known from the [device_frame](https://pub.dev/packages/device_frame) package |  |
+
+## Builders
+
+`Widgetbook` exposes various builder functions to allow customization of how `WidgetbookUseCase`s are displayed.
+
+### `deviceFrameBuilder`
+
+The `deviceFrameBuilder` in combination with the `frames` property can be used to add your or an existing implementation of a device frame: 
+
+For the [device_frame](https://pub.dev/packages/device_frame) package this can look like this:
+
+```dart
+Widgetbook.material(
+  deviceFrameBuilder: (context, device, renderMode, child) {
+    if (renderMode == DeviceFrame.deviceFrame()) {
+      return frame.DeviceFrame(
+        device: frame.Devices.ios.iPhone12,
+        screen: child,
+      );
+    }
+
+    // default to no device frame
+    return child;
+  },
+)
+```
+
+### `localizationBuilder`
+
+The default of `localizationBuilder` is defined as: 
+
+```dart
+(
+  BuildContext context,
+  List<Locale> supportedLocales,
+  List<LocalizationsDelegate<dynamic>>? localizationsDelegates,
+  Locale activeLocale,
+  Widget child,
+) {
+  if (localizationsDelegates != null) {
+    return Localizations(
+      locale: activeLocale,
+      delegates: localizationsDelegates,
+      child: child,
+    );
+  }
+
+  return child;
+};
+```
+
+### `themeBuilder`
+
+The `themeBuilder` allows you to inject theme data into the `Widget` tree. An implementation for `CupertinoThemeData` could look like this:
+
+```dart
+Widgetbook<CupertinoThemeData>(
+  themeBuilder: (
+    BuildContext context,
+    CupertinoThemeData theme,
+    Widget child,
+  ) {
+    return CupertinoTheme(
+      data: theme,
+      child: child,
+    );
+  },
+)
+```
+
+### `scaffoldBuilder` and `useCaseBuilder`
+
+Both the `scaffoldBuilder` and `useCaseBuilder` can be used to wrap the `Widget` with e.g. a `Scaffold` or some other `Widget` like a `Center`, `Container` or `Padding`.  
+
```

Comment:
# Widgetbook Cloud

Our early users have been wishing for a solution to easily share their Widgetbook with their designers, managers and clients to simplify review processes. Therefore, the Widgetbook Team has been developing the collaboration platform 'Widgetbook Cloud'. It hosts different branches of your Widgetbook and lets your team review your work in seconds in a password protected environment. Check out our website (link to website) to find a short demo video and more information.